qsirecon/workflows/base.py

```diff
@@ -355,10 +355,31 @@ def _load_recon_spec(spec_name):
             spec = json.load(f)
         except Exception:
             raise Exception("Unable to read JSON spec. Check the syntax.")
+
     if config.execution.sloppy:
         config.loggers.workflow.warning("Forcing reconstruction to use unrealistic parameters")
         spec = make_sloppy(spec)
 
+    # Expand any "scalars_from" lists into separate nodes
+    orig_spec = spec.copy()
+    spec["nodes"] = []
+    for node in orig_spec["nodes"]:
+        if "scalars_from" in node.keys() and isinstance(node["scalars_from"], list):
+            for scalar_source in node["scalars_from"]:
+                new_node = node.copy()
+                new_node["name"] = f"{node['name']}_{scalar_source}"
+                new_node["scalars_from"] = scalar_source
+                if "qsirecon_suffix" not in new_node.keys():
+                    # Infer the suffix from the source node
+                    for temp_node in spec["nodes"]:
+                        if temp_node["name"] == scalar_source:
+                            new_node["qsirecon_suffix"] = temp_node["qsirecon_suffix"]
+                            continue
+
+                spec["nodes"].append(new_node)
+        else:
+            spec["nodes"].append(node)
```

Comment:
@mattcieslak check this out. WDYT?

Reply:
wow this is way prettier than I thought it would have to be

Reply:
As long as nodes with `scalars_from` are endpoints in the workflow (i.e., don't feed into other nodes), I think it should work. If they do feed into others then I might need to make it a bit more complicated.

Reply:
does .copy() work here? I remember needing to deepcopy nipype objects back in the day

Reply:
It should since this is just a dictionary at the moment.

Reply:
Switched to deepcopy to be safe given our convo in scrum.